### ISSUE TITLE
Add SolderPAD 2.0

### ISSUE THREAD

**Issue by FedericoVaga** [spdx/license-list-XML#945]
Among the SPDX licenses there are SHL-0.5 and SHL-0.51. Since 2018 there is also SHL-2.0 (https://solderpad.org/licenses/SHL-2.0/).  Is it possible to add it to the list of recognized licences?

**Comment by jlovejoy** [comment]
yes, that has been the plan, but I was waiting on some feedback from the license author. will try to chase!

**Comment by swinslow** [comment]
Hi @jlovejoy, did you get the feedback you needed from the license author for SolderPAD 2.0? Or is this one that we should move out past 3.9?

**Comment by swinslow** [comment]
Discussed on 2020-04-09 legal team call, @jlovejoy will follow up with license author.

**Comment by jlovejoy** [comment]
discussed on legal call 23rd April - this is written to amend Apach-2.0 so thus should be added on the Exceptions List b/c one has to provide full text of Apache-2.0 with it, so short identifer would be:
   Apache-2.0 WITH SHL-2.0

2.1 is coming also - will deal with both at same time

**Comment by jlovejoy** [comment]
license is in process of moving to have FOSSI Foundation to be steward - want to have them bless the decision to add and short identifier

**Comment by andrewjskatz** [comment]
I've just had a meeting with FOSSi, and they have agreed to go ahead with stewardship of Solderpad, and we have been working on finalising the reference text of both Solderpad 2.0 and Solderpad 2.1.

**Comment by jlovejoy** [comment]
@andrewjskatz - I think we should have both 2.1 and 2.0 no?
2.0 is already in use, isn't that true? and we already have the set text for it, so easy add?

**Comment by jlovejoy** [comment]
oops, this is not tagged as accepted to add - I thought for sure we discussed it some time ago and the pause was due to license steward change and see about 2.1 version
@swinslow  - do you recall this?

**Comment by swinslow** [comment]
@jlovejoy yes, I think we had discussed on the call and were comfortable with adding, but it was pending confirmation on the FOSSi / license steward question. Since we know now that they're in favor I think we can mark this as accepted (i'll do so now).

**Comment by jlovejoy** [comment]
so, this needs to be an exception in same way as 2.1
see #1029

not sure how to accommodate bold/italic text though

**Comment by andrewjskatz** [comment]
@jlovejoy you can see why this is a reason we have removed the formatting in version 2.1. I was inspired by the canonical version of Apache 2.0 does contain formatting https://www.apache.org/licenses/LICENSE-2.0 but I can in no way blame ASF for my own idiocy.

There are two options I can propose. I can either mark up the formatting as *bold* and +italic+ - I'm happy to provide a file to do that (but I'm slightly worried that in some circumstances the * and + might be interpreted as break characters and cause odd things to happen in text editors or IDEs, or I would alter the preamble so it reads:

If your copy of this document has some words in italics, these are intended indicate changes from the Apache License, but are indicative and not to be taken into account in interpretation.

(originally  

Words in italics indicate changes rom the Apache License, but are indicative and not to be taken into account in interpretation.
)

And then I will insert quotes around the emboldened definitions. 

Neither of these changes have any effect on the legal meaning of the document, so I'm reasonably comfortable making them.

**Comment by swinslow** [comment]
@jlovejoy @andrewjskatz I'd suggest we just merge #1029 as-is, with the license in plain text since that's the only thing the license list can render.

And then (maybe as a follow-up PR) we add to the notes for SHL-2.0 a statement making it clear that the italics referenced in the license cannot be displayed in the version on the license list, but can be seen on the license steward's site. Would you be okay with that?

**Comment by swinslow** [comment]
Merged in #1029 and #1033, thanks all!
